python/perspective/perspective/src/table.cpp

```diff
@@ -193,6 +194,10 @@ std::shared_ptr<Table> make_table_py(t_val table, t_data_accessor accessor,
         _fill_data(data_table, accessor, input_schema, index, offset, limit, is_update);
     }
 
+    if (is_arrow) {
```

Comment:
is there any case where `is_arrow && is_delete`?

Reply:
e.g. should this match above `    if (is_arrow && !is_delete) {`

Reply:
There shouldn't be any - remove only takes a list of pkeys, but this whole conditional block is kind of messy. I'll take a look and fix.

Reply:
This logic is incorrect, but so is the fix suggestion - there should be (or will need to be) one case, for `remove()` ... when we get around to fixing this API method generally ...